apps/client/src/app/lobby/[roomId]/page.tsx

```diff
@@ -0,0 +1,178 @@
+'use client';
+import Image from 'next/image';
```

Comment:
Can we do this for easiness to read?
```suggestion
'use client';

import Image from 'next/image';
```